planner/core/task.go

```diff
@@ -1060,6 +1060,7 @@ func setTableScanToTableRowIDScan(p PhysicalPlan) {
 type rootTask struct {
 	p   PhysicalPlan
 	cst float64
+	isEmpty bool
```

Comment:
This should be a temp solution? What would it be after tiflash support dual or we eliminate the dual before converting the task?

Reply:
If this is a tmp solution. We can add a TODO here and comment on its usage.

Reply:
This not a "tmp" solution because we are using this flag to check if this task returns empty results. It helps to simplify the plan as much as possible and doesn't conflict with supporting of dual table on TiFlash.